CHANGELOG.md

```diff
@@ -19,7 +19,7 @@
 
 ## New Functionality
 
--
+- [OData] Support for function imports with and without parameters using the OData V4 syntax.
```

Comment:
Although the whole feature was missing, to be fair, we announced that the v4 function/action was completed (OKR), we might want to move this line to bug fix.

Reply:
Okay, I understand that if this was announced already, then it should be counted as a bug fix. Wasn't aware.